atomicapp/nulecule/container.py

```diff
@@ -155,9 +155,12 @@ def extract(self, image, source, dest, update=False):
 
         # Copy files
         logger.debug('Copying nulecule data from %s to %s' % (src, dest))
-        Utils.copy_dir(src, dest, update)
-        logger.debug('Removing tmp dir: %s' % tmpdir)
-        Utils.rm_dir(tmpdir)
+        if file:
+            Utils.copy_file(src, dest)
+        else:
+            Utils.copy_dir(src, dest, update)
```

Comment:
the two statements below need to be unindented. the tmpdir needs to be deleted regardless of if we were doing just a file copy or an entire dir copy.